### ISSUE TITLE
ReactCookieProps type definition might be incorrect?

### ISSUE THREAD

**Issue by apfrancis** [reactivestack/cookies#167]
Hi, I've been using this package for a while now and it's been great. Thanks for working on it 👍 

Just tried to upgrade to the latest version (from 2.2.0) and seeing a Typescript compilation error.

```
TS2322: Type '{ children: Element; }' is not assignable to type 'Readonly<ReactCookieProps>'.
  Property 'cookies' is missing in type '{ children: Element; }'
```

I'm using the `<CookiesProvider>` in a React component that is rendered on the client and my understanding was that the the `cookies` prop wasn't required in that context. 

If modify `types.d.ts` and make `cookies` optional it looks like my code doesn't break. Just wanted to check whether this was a change to the API or an error in the type definition. 

Thanks!

**Comment by eXon** [comment]
My mistake, I'm not used to TypeScript for HOC. This will be fixed in the next version.

**Comment by apfrancis** [comment]
Great, thanks 🎉

**Comment by eXon** [comment]
@apfrancis Can you confirm your issue has been fixed in `v3.0.4`? Thank you!

**Comment by apfrancis** [comment]
@eXon yes that fixes the compilation errors, thanks!
